students/yesol/westagram/users/views.py

```diff
@@ -9,22 +9,27 @@ class MembersRegisterView(View):
     def post(self, request):
         try:
             data = json.loads(request.body)
+
+            email          = data["email"]
+            input_password = data["password"]
```

Comment:
```suggestion
            password = data["password"]
```
password로도 충분할 것 같습니다.